static/bidder-info/mgidX.yaml

```diff
@@ -1,4 +1,7 @@
-endpoint: "https://us-east-x.mgid.com/pserver"
+disabled: true
+# We have the following regional endpoint domains: 'us-east-x' and 'eu'
+# Please deploy this config in each of your datacenters with the appropriate regional subdomain
+endpoint: "https://REGION.mgid.com/pserver"
```

Comment:
Are you sure you want to change the default to disabled? This may cause your adapter to not be enabled by some hosts if they miss this change.